vendor/redhat/labels.md

```diff
@@ -4,7 +4,7 @@ Red Hat built images will include a fields in the metadata to identify the offic
 
 ## Naming and versioning related labels
 
-The Fields, all required, are:
+The Fields, all required even the other labels:
```

Comment:
This is not a sentence.  Are you saying 
These fields and related labels are all required?